### ISSUE TITLE
Be able to use a single Label

### ISSUE THREAD

**Issue by Colanim** [heartexlabs/label-studio#375]
I'm doing extractive text summarization.

So given a text, I want to label the salient sentences.

I tried :

```
<Labels name="ext" toName="text">
   <Label value="Salient sentence" />
</Labels>
```

But it yields the error :

```
Validation Failed On View/Labels: OrderedDict([('Value', 'Salient sentence')]) Is Not Of Type 'Array'
```

I think it should be possible to have a single label.

**Comment by makseq** [comment]
We are working on this issue, I think we will include the fix in release/0.7.4.

**Comment by makseq** [comment]
@Colanim I fixed this issue in https://github.com/heartexlabs/label-studio/pull/350. Please, try it if you can.
